pootle/settings/60-translation.conf

```diff
@@ -51,6 +51,9 @@ POOTLE_MT_BACKENDS = [
 # the trailing slash.
 AMAGAMA_URL = 'https://amagama-live.translatehouse.org/api/v1/'
 
+# List of available source languages in AMAGAMA
+AMAGAMA_SOURCE_LANGUAGES = ('en', 'en_US', 'en-US')
```

Comment:
Is not a problem to specify unsupported sources languages though, just raising that default setting is probably better to reflect what is supported by the main amaGama server.